tasks/facial_landmark_detection/README.md

```diff
@@ -57,6 +57,16 @@ The trained models will be saved in the models directory. You can modify the out
 The training using the parameters in train_candidate_models.py produces another CSV file (search_results_with_full_validation_error.csv) with validation error data added from the training. The following graph is produced with such data:
 ![Results of full training](Full_Training_Validation_Error_vs_onnx_latency_(ms).png)
 
+## Quantization Aware Training
+To perform QAT on a selected model, use the --qat option which utilizes the quantizable version of the model to perform the training. 
+If needed for higher accuracy, use the --qat_skip_layers <# of layers> to partially quantize the model with specified number of layers left in floating point.
```

Comment:
Do we need to specify the direction of skipping? I assume the skipping starts from the head of the model and go towards the stem.

Reply:
Fixed.